src/styles/stars.scss

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+  margin: 0;
+  line-height: 16px;
+}
+
+.stars__star {
+  box-sizing: border-box;
+  list-style-type: none;
+  background: url(../images/star.svg) no-repeat center center;
+  background-position: center;
+  background-repeat: no-repeat;
```

Comment:
The `background` shorthand property already includes `background-position` and `background-repeat`. There's no need to repeat these properties on lines 13 and 14.